content/shmem_init.tex

```diff
@@ -33,13 +33,13 @@
 }      
 
 \apinotes{
-    As of \openshmem Specification 1.2 the use of \FUNC{start\_pes} has been
+    As of \openshmem Specification 1.2, the use of \FUNC{start\_pes} has been
```

Comment:
For consistency, `\openshmem Specification 1.2` should be changed to `\openshmem[1.2]`, but I've also noted this in #110.

Reply:
Updated in #110.